### ISSUE TITLE
Leetcode Update Compatibility

### ISSUE THREAD

**Issue by aavin95** [QasimWani/LeetHub#404]
I'm not sure if other people are having this problem but I just noticed that since the new LeetCode update came out my LeetHub files aren't updating when I have a new solution or when I solve a new problem. Thank you for this amazing service you provide!

**Comment by KHemanthRaju** [comment]
Yes, Even I am facing the same issue. Could you pls resolve asap.

**Comment by debopamabasu** [comment]
Same here...

**Comment by h-attari** [comment]
Yes, I am also facing the same issue.

**Comment by r4deadv** [comment]
I'm facing the same issue. Will it be an update out soon?
